### DISCUSSION TITLE
How to create a dashboard (filter, wiget) to show results for 5 Teams using Specflow+Xunit project

### DISCUSSION THREAD

**Discussion by sureshp191** [reportportal/reportportal#1600] [Q&A] [answered]
I am trying to create a dashboard that shows the Test Progress (Pass, Fail etc) for 5 teams. ReportPortal.config.json has been configured so that the schema, server {url, project} has the same values like "project": "XYZ Regression Status" but name in launch has different value like "name": "Team 1 Regression" (likewise for T"Team 2 Regression" and so on), similarly for attributes "attributes": [ "Regression Suite", "Team 1" ] so on. 

I am able to run the automation and see the results on the Report Portal launch for different teams. I created a filter with name, attribute value and time (dynamic) for each team, example Team1 Daily Regression Status, Team2 Daily Regression Status so on. 

On the existing dashboard, which widget I can use so that on the Y-axis I have the team's name  (Team1, Team 2 etc) and X-axis I have the test case counts and the Bar Chart shows the test result - Pass, Fail etc for each team? 

As I have 5 filters, one for each team, with one widget is it possible to select all the 5 filters? if not please advise a work around.

**Comment by evjlobanova** [comment, marked as answer]
You can use Cumulative trend Chart, Component Health Check Widget or Component Health Check widget (table view).
Please find instructions how to create them 
https://reportportal.io/docs/Cumulative-trend-chart
https://reportportal.io/docs/Component-health-check
https://reportportal.io/docs/Table-Component-health

For all of these widgets you should have attributes in the format 'key: value' 
for example 'project: Team 1' 'scope: Regression'
